### DISCUSSION TITLE
multi tenant

### DISCUSSION THREAD

**Discussion by yxzhou1212** [linkerd/linkerd2#8146] [General] [answered]
Linkerd2

Does it support multi tenant deployment schemes?

**Comment by wmorgan** [comment, marked as answer]
You can use Linkerd with multi-tenant deployments. The Linkerd control plane itself is cluster-wide. However, Linkerd respects all Kubernetes RBAC rules and other tenancy mechanisms, and also provides features (like [locking down traffic between namespaces](https://buoyant.io/2021/12/14/locking-down-network-traffic-between-kubernetes-namespaces/)) that are useful in multi-tenant deployments.

**Reply by yxzhou1212** [reply]
thanks for the response! I have deployed on my k8s cluster。
